app/image/image.py

```diff
@@ -0,0 +1,44 @@
+"""Support for containing image."""
+
+import numpy
+
+
+class RawDataContainer:
+    """Structure for file data and communication."""
+    def __init__(self, data_buffer):
+        self.data_buffer = data_buffer
+
+    @classmethod
+    def from_file(cls, file_path):
+
+        data_buffer = ""
+
+        try:
+            with open(file_path, "rb") as file:
+                data_buffer = file.read()
+                return cls(data_buffer)
+
+        except EnvironmentError as err:
+            raise Exception(
+                "Error occured while trying to read from file {}".format(
```

Comment:
please print the except reason. This may fail for many different reasons - file does not exist, permission etc.